tuftool/README.md

```diff
@@ -1,5 +1,11 @@
 **tuftool** is a Rust command-line utility for generating and signing TUF repositories.
 
+## Dependencies
+
+Make sure you have the following dependencies installed before installing `tuftool`:
```

Comment:
I would tweak the ending a bit to be _"installed before **building** `tuftool`"_